### ISSUE TITLE
Track down sources of EFI images

### ISSUE THREAD

**Issue by sundarnagarajan** [sundarnagarajan/rdp-thinbook-linux#6]
Files under remaster/iso_post/efi/EFI/BOOT:
bootia32.efi
BOOTx64.EFI
grubx64.efi

These are from linuxium. I would PREFER to create t hem on the fly using grub-mkbootimage
There must be sources (of the config used) somewhere in the sources for Ubuntu

**Comment by sundarnagarajan** [comment]
From [this discussion on superuser.com](https://superuser.com/questions/1153247/creating-efi-boot-for-an-already-installed-linux/1153283#1153283) it appears that I need to use ```grub-mkimage``` with the ```-p ""``` option

cd ```<grub2_compiled_source_dir>/grub-core```
```../grub-mkimage -O ${EFI_ARCH}-efi -d . -o grub.efi -p "" part_gpt part_msdos ntfs ntfscomp hfsplus fat ext2 normal chain boot configfile linux multiboot```
Note : The ```-p ""``` option is important for creating a portable grub.efi app.


Also see this [github repo on nstructions to create Ubuntu 16.04 LTS install media for ASUS EeeBook X205TA](https://github.com/lopaka/instructions/blob/master/ubuntu-16.04-install-asus-x205ta.md):

```
    ../grub-mkimage -d . -o bootia32.efi -O i386-efi -p /boot/grub \
   ntfs hfs appleldr boot cat efi_gop efi_uga elf fat hfsplus iso9660 linux keylayouts \
   memdisk minicmd part_apple ext2 extcmd xfs xnu part_bsd part_gpt search search_fs_file \
   chain btrfs loadbios loadenv lvm minix minix2 reiserfs memrw mmap msdospart scsi loopback \
   normal configfile gzio all_video efi_gop efi_uga gfxterm gettext echo boot chain eval
```
And another [github repo on 32-bit efi for Asus T100](https://github.com/jfwells/linux-asus-t100ta/tree/master/boot):

```
$ ../grub-mkimage -d . -o bootia32.efi -O i386-efi -p /boot/grub ntfs hfs appleldr boot cat efi_gop efi_uga elf fat hfsplus iso9660 linux keylayouts memdisk minicmd part_apple ext2 extcmd xfs xnu part_bsd part_gpt search search_fs_file chain btrfs loadbios loadenv lvm minix minix2 reiserfs memrw mmap msdospart scsi loopback normal configfile gzio all_video efi_gop efi_uga gfxterm gettext echo boot chain eval
```

**Comment by sundarnagarajan** [comment]
Closed with [commit 5745da56e82ee4a0276a096126ae59f9e0c5a7bd](https://github.com/sundarnagarajan/rdp-thinbook-linux/commit/5745da56e82ee4a0276a096126ae59f9e0c5a7bd)

Thanks to @lopaka and @jfwells
